index.bs

```diff
@@ -874,6 +832,21 @@ These number values are generally the preferred values to use for 'line-height'
 because they inherit better than length values.
 </div>
 
+Issue: The paragraph below is hard to understand.
```

Comment:
How about making this a clear recommendation to err on the side of reduced complexity? This is kind of implied but not explicitly stated. Something like “When making decisions about whether dependencies should affect computed values, try to not add undue complexity.“ ?